cdap-ui/package.json

```diff
@@ -171,7 +170,7 @@
     "font-awesome": "4.6.3",
     "frameguard": "3.1.0",
     "fuse.js": "2.5.0",
-    "global": "^4.4.0",
+    "global": "4.4.0",
```

Comment:
nit: I am actually removing this in another branch. This is not a package we use anywhere. Someone (possibly me) must have typed `yarn add global <some-package-name>` which must have add this as a module.